### DISCUSSION TITLE
tel and callto protocols don't work as HTML elements in markdown files.

### DISCUSSION THREAD

**Discussion by FabioFlorey** [community/community#43350] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello!👋
I was writing a resume on github and I wanted to write a markdown version of it, I planned to insert a clickable images for contacts when I noticed that I couldn't use some protocols, tel and callto in particular. This seems intentional but I don't see the point since other communication protocols are preserved.

#### Test with phone number 

<br> `tel:` protocol

<a href="tel:+390000000000">test w/ phone number</a>

```markdown
<a href="tel:+390000000000">test w/ phone number</a>
```

`callto:` protocol

<a href="callto:+390000000000">test w/ phone number</a>

```markdown
<a href="callto:+390000000000">test w/ phone number</a>
```

In both cases the hyperlink is completely gone, but the text remains.

link with `tel:` protocol wrapping an image:

<a href="tel:+390000000000"><img src="https://www.rd.com/wp-content/uploads/2018/02/30_Adorable-Puppy-Pictures-that-Will-Make-You-Melt_124167640_YamabikaY.jpg"></a>

```markdown
<a href="tel:+390000000000"><img src="./puppy.png"></a>
```
As expected, the image is depicted but the hyperlink is scraped. This doesn't work for communication protocols such as `mailto`

<a href="mailto:example@example.org?subject=This is an example">example</a>

```markdown
<a href="mailto:example@example.org?subject=This is an example">example</a>
```

### Expected results

I expected the procols to work and to start a call or skype on-click

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FabioFlorey ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
